src/components/ActionCard/sgds-action-card.ts

```diff
@@ -83,25 +117,20 @@ export class SgdsActionCard extends CardElement {
           ["is-active"]: this.active
         })}
         "
+        part="base"
       >
-        <div class="card-body">
-          <h6 class="text-muted card-subtitle">
+        <div class="card-body" part="body">
+          <h6 class="text-muted card-subtitle" part="subtitle">
             <div>
-              ${this.iconName == undefined ? undefined : icon}
+              <slot name="icon"></slot>
               <slot name="card-subtitle"></slot></slot>
```

Comment:
Theres an extra slot closure here
```suggestion
              <slot name="card-subtitle"></slot>
```